src/test/integration/user/email.test.ts

```diff
@@ -0,0 +1,68 @@
+import { expect } from "chai";
+import sinon from "sinon";
+import { EmailService } from "@core/email/email.service";
+
+describe("Teste de email", () => {
+  let emailService: EmailService;
+  let resendMock: { emails: { send: sinon.SinonStub } };
+
+  beforeEach(() => {
+    resendMock = {
+      emails: {
+        send: sinon.stub(),
+      },
+    };
+
+    emailService = new EmailService();
+
+    (emailService as any).resendClient = resendMock;
+  });
+
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  it("Deve enviar um email com sucesso", async () => {
+    (resendMock.emails.send as sinon.SinonStub).resolves({
+      id: "123",
+      error: null,
+    });
+
+    const result = await emailService.sendEmail({
+      from: "no-reply@guina.dev",
+      to: "user@example.com",
+      subject: "Teste",
+      text: "Corpo do e-mail",
+    });
+
+    expect(result.message).to.equal("Email enviado com sucesso!");
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+    expect(resendMock.emails.send.calledOnce).to.be.true;
```

Comment:
Evite ignorar as regras do lint

```suggestion
    expect(resendMock.emails.send.calledOnce).to.be.equal(true);
```